packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5453,19 +5467,20 @@ export default class Meeting extends StatelessWebexPlugin {
           this.setNetworkStatus(NETWORK_STATUS.CONNECTED);
           this.reconnectionManager.iceReconnected();
           this.statsAnalyzer.startAnalyzer(this.mediaProperties.webrtcMediaConnection);
+          this.hasMediaConnectionConnectedAtLeastOnce = true;
```

Comment:
when you rebase to latest beta, you'll see that I had to do something similar for setting the right icePhase when we send 'client.ice.end' when we get `ConnectionState.Connected`. I've managed to avoid adding a new flag by using the `Meeting.networkStatus`

Reply:
Nice I hadn't thought of using that. I've made this change